cli/src/main.rs

```diff
@@ -1,3 +1,4 @@
+use log::{debug, error, info, warn};
```

Comment:
I can't comment on the whole block, but the three sections of imports here are following a Java style standard. The first block is for imports provided by the language itself and do not need to be acquired through a package manager. The second block are imports that are dependencies of the project provided by the package manager, and the third block of imports are other modules within the same project.

So, can you move this line to the second set of imports and insert it in a sorted order in that block?

Reply:
(The value in this is it makes it easier to determine where to find the documentation, or the source code itself, just by seeing which block the import line is grouped in.)

Reply:
Sure, will do and take it into account for the future.